package.json

```diff
@@ -89,7 +89,7 @@
     "react-spring": "^9.3.0",
     "react-transition-group": "^4.4.2",
     "react-waypoint": "^10.1.0",
-    "sanitize-html": "^2.6.1",
+    "sanitize-html": "2.6.1",
```

Comment:
why?

Reply:
2.8 and higher version of `sanitize-html` breaks dev build, I fixed working version.